cdap-app-templates/cdap-etl/cdap-etl-realtime/pom.xml

```diff
@@ -59,6 +59,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
+      <scope>test</scope>
```

Comment:
This should be set in dependencyManagement in the parent, especially as for junit, which is almost always never used other than in test scope.